client/components/plans/plan-actions/style.scss

```diff
@@ -171,3 +171,7 @@
 .jetpack_business.plan-actions__illustration {
 	background-image: url('/calypso/images/plans/plan-business.svg');
 }
+
+.jetpack_premium .is-image-button, .jetpack_business .is-image-button {
+	display:none;
```

Comment:
The intention is to bring the images back -- we just haven't had time work on them yet. So I figured better to hide them temporarily. LMK if you disagree.